src/libstore/nar-info-disk-cache.cc

```diff
@@ -166,19 +179,33 @@ class NarInfoDiskCacheImpl : public NarInfoDiskCache
         return i->second;
     }
 
+private:
+
+    bool cacheIsUpToDate(const Cache &cache) {
+        return cache.lastCacheInfoFetch > (time(0) - NarInfoDiskCacheImpl::cacheInfoTtl);
```

Comment:
```suggestion
        return cache.lastCacheInfoFetch > (time(0) - cacheInfoTtl);
```

(I think, correct me if we're not in the right context)